admin/admin-filters-columns.php

```diff
@@ -341,6 +335,21 @@ public function term_column( $out, $column, $term_id ) {
 			$out .= $this->links->new_term_translation_link( $term_id, $taxonomy, $post_type, $language );
 		}
 
+		if ( $column == $this->get_first_language_column() ) {
```

Comment:
Since we're editing this part...

```suggestion
		if ( $this->get_first_language_column() === $column ) {
```